lib/meataxe.gi

```diff
@@ -1505,24 +1434,20 @@ end;
 ## It is based on code of MinPolCoeffsMat.
 ## The optional third argument is for returning the basis for this block.
```

Comment:
Apparently not the third but the fourth argument ... (More is wrong in this documentation.)